pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -350,15 +350,17 @@ func verifyRegistriesConfigAndPolicyJSONContents(t *testing.T, mc *mcfgv1.Machin
 		imgcfg.Spec.RegistrySources.BlockedRegistries, icsps)
 	require.NoError(t, err)
 	assert.Equal(t, mcName, mc.ObjectMeta.Name)
+
+	ignCfg := ctrlcommon.DecodeIgnitionConfigSpecV2OrDie(mc.Spec.Config.Raw)
```

Comment:
Maybe we should be testing MC objects here still and do some extra conversion instead of testing ignCfg directly

Reply:
Hm looking closer at this, I'm not sure I follow here. The func still takes an MC as arg, and we need to look at the decoded config to be able to tell what's inside. Happy to change it up if you could elaborate a little more tho :)

Reply:
I think I confused something. This is good as is